src/containers/DeleteAccountPage/DeleteAccountForm/DeleteAccountForm.module.css

```diff
@@ -0,0 +1,146 @@
+@import '../../../styles/customMediaQueries.css';
+
+.root {
+}
+
+/* Shared styles for infoText variants */
+.infoText {
+  composes: textSmall from global;
+
+  display: inline-block;
+  padding-left: 18px;
+  background-repeat: no-repeat;
+  background-position: top 4px left;
+
+  margin-top: 11px;
+  margin-bottom: 0;
+
+  @media (--viewportMedium) {
+    margin-top: 14px;
+    margin-bottom: 1px;
+  }
+}
+.deleteAccountSection {
+  margin-top: 16px;
+  margin-bottom: 46px;
+  padding-top: 6px;
+
+  @media (--viewportMedium) {
+    margin-bottom: 56px;
+    padding-top: 0;
+  }
+}
+
+.phone {
```

Comment:
Remember to clean this file so that it only contains classes that are actually in use.